app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>User</h1>
+<%= render partial: "username", locals: { user: @user } %>
+<div>
+<h3>Bio</h3>
+<p><%=@user.bio%></p>
+</div>
+<ul class='user-posts'>
+<% @user.posts.each_with_index do |post, index| %>
```

Comment:
- [ ] Please use the custom method `three_recent_posts` here that you created to display only three posts instead of displaying all as requested in the project requirements.